_includes/release-notes/packages.md

```diff
@@ -0,0 +1,43 @@
+## Magento packages
```

Comment:
We need some introductory text before this heading, otherwise, we end up with the H1 topic title immediately followed by an H2 heading. It's bad practice.

Reply:
agree